pkg/commands/kvstore/create.go

```diff
@@ -28,6 +31,7 @@ func NewCreateCommand(parent argparser.Registerer, g *global.Data) *CreateComman
 	}
 	c.CmdClause = parent.Command("create", "Create a KV Store")
 	c.CmdClause.Flag("name", "Name of KV Store").Short('n').Required().StringVar(&c.Input.Name)
+	c.CmdClause.Flag("location", "Regional location of KV Store").Short('l').HintOptions(locations...).EnumVar(&c.Input.Location, locations...)
```

Comment:
Please move this to sit above `name` flag so the help output shows flags in sorted order (this is something I've been doing wherever/whenever possible and I'd like it to be as consistent as possible moving forward).

**EDIT:** So with that in mind I see `--json` flag will need reordering too.